diagnostic.md

```diff
@@ -73,20 +93,46 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --request POST http://www.example.com/donuts \
+--header "Content-Type: application/json" \
+--data '{"name": "French Cruller",
+"price": "$0.99"}'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function() {
+  let item = new donut;
+$.ajax({
+  url: 'http://www.example.com/donuts/',
+  type 'POST',
+  contentType: false,
+  processData: false,
+  data: item
+})
+.done(logResponseBody)
+.fail(logRequestError);
+};
 ```
```

Comment:
I know what you wanted to do,  and I think you have the right idea, we were looking for something like this here thought.  I won't take any points off because I see you did the right thing in the curl request:
```
let createDonut = function (event) {
  event.preventDefault();
  $.ajax({
    url: 'http://www.example.com/donuts',
    type: 'POST',
    data: {
      name: 'French Cruller',
      price: '$0.99'
    }
  })
  .done(logResponseBody)
  .fail(logRequestError);
};
```